fontgenerator.js

```diff
@@ -144,7 +142,7 @@ TinyFontGenerator = {
     if(this.OS2data!==false) { return this.OS2data; }
 
     var data = "";
-    data += " 00 04";  // The current (Jan 2012) OS/2 table version is 4
+    data += " 00 01";  // The OS/2 table version is 1 (TrueType rev 1.66)
```

Comment:
Unfortunately, this breaks loading in several browsers that refuse to acknowledge OS/2 v1 is real.